homeassistant/components/sensor/postnl.py

```diff
@@ -50,18 +52,36 @@ def setup_platform(hass, config, add_devices, discovery_info=None):
         _LOGGER.exception("Can't connect to the PostNL webservice")
         return
 
-    add_devices([PostNLSensor(api, name)], True)
+    sensor_items = []
+    sensor_items.extend([PostNLSensor(
+        hass,
+        'PostNL-package',
```

Comment:
`PostNL` is the part of the default name. The sensor is called `package` or `packages`. Create the name later while you initialize the sensor with something like `self._name = '{} {}'.format(name, sensor_type)` while `name` is the default name or the one that was defined by the user.